src/components/Table/index.js

```diff
@@ -525,4 +535,35 @@ const Table = () => {
   );
 };
 
+const ProgressBar = ({ name, value, total, className, barClassName }) => {
+  return (
+    <div>
```

Comment:
Tiny nit: This can be a React component, ie. `<>`

Reply:
I kept it since css is using it to add margins between the sections, but I can change the CSS selectors if that's preferred.